selective_upcase and selective_tag filters: https://github.com/metano…

opoudjis opened the pull request:
…rma/mn-samples-icc/issues/36

opoudjis committed:
selective_upcase and selective_tag filters: https://github.com/metanorma/mn-samples-icc/issues/36

opoudjis committed:
selective_upcase and selective_tag filters: https://github.com/metanorma/mn-samples-icc/issues/36

ronaldtse commented on _specs/relaton-render.adoc:
```diff
@@ -2449,7 +2449,35 @@ lowercase other words in the string.
+"BLACK and DECKER"
+====
+
+==== Selective Tag (`selective_tag`)
```
This introduces XML elements into the output of relaton-render. Do we support this? This means relaton-render cannot be used in any non MN XML use case.

opoudjis commented:
We do support it, but XML is only in the templates, not in the variables, so we can support non-XML markup with alternate templates. Will add illustration of that to documentation and to rspec of relaton-render.

opoudjis committed:
XML markup discussion in Relaton-Render

opoudjis committed:
XML markup discussion in Relaton-Render

opoudjis committed:
updated relaton-render: https://github.com/metanorma/metanorma-jis/issues/388

opoudjis committed:
updated relaton-render: https://github.com/metanorma/metanorma-jis/issues/388

opoudjis committed:
config parameter: https://github.com/metanorma/metanorma-standoc/issues/1125

opoudjis committed:
config parameter: https://github.com/metanorma/metanorma-standoc/issues/1125

opoudjis committed:
rename Preliminary Draft, Revision Draft to Predecessor Draft, Successor Draft: https://github.com/relaton/relaton.org/issues/77

opoudjis committed:
rename Preliminary Draft, Revision Draft to Predecessor Draft, Successor Draft: https://github.com/relaton/relaton.org/issues/77

opoudjis commented:
@ronaldtse These updates need to be committed.

opoudjis committed:
disambiguated_date in relaton-render: https://github.com/metanorma/metanorma-iso/issues/1529

opoudjis committed:
disambiguated_date in relaton-render: https://github.com/metanorma/metanorma-iso/issues/1529

opoudjis committed:
disambiguated_date in relaton-render: https://github.com/metanorma/metanorma-iso/issues/1529

opoudjis committed:
disambiguated_date in relaton-render: https://github.com/metanorma/metanorma-iso/issues/1529

ronaldtse committed:
migrate PR changes to VitePress structure

- model/relations.md: rename preliminaryDraftOf/revisionDraftOf to predecessorDraftOf/successorDraftOf
- specs/relaton-render.md: comprehensive content migration including
  - new data fields (place_raw, status_raw, disambiguated_date)
  - new Liquid filters (selective_upcase, selective_tag)
  - Citation templates (citetemplate) section
  - updated field descriptions and labels (punct/*, upcase, etc.)
  - various text clarifications across the document
